test/framework/dump.go

```diff
@@ -173,8 +173,15 @@ func (f *CommonFramework) DumpLogsForPodsWithLabelsInNamespace(ctx context.Conte
 
 	var result error
 	for _, pod := range pods.Items {
-		if err := f.DumpLogsForPodInNamespace(ctx, k8sClient, namespace, pod.Name, &corev1.PodLogOptions{}); err != nil {
-			result = multierror.Append(result, err)
+		for _, container := range pod.Spec.InitContainers {
+			if err := f.DumpLogsForPodInNamespace(ctx, k8sClient, namespace, pod.Name, &corev1.PodLogOptions{Container: container.Name}); err != nil {
```

Comment:
WDYT about enhancing `DumpLogsForPodInNamespace` to log out the container name in the beginning or does this happen already implicitly?

Reply:
makes sense 👍

Reply:
makes sense 👍